Assets/Scripts/Framework/Buttons/MainMenu/QuitButton.cs

```diff
@@ -0,0 +1,12 @@
+using UnityEngine;
+
+namespace DerailedDeliveries.Framework.Buttons.MainMenu
+{
+    /// <summary>
+    /// A <see cref="BasicButton"/> class which is responsible for closing the application.
```

Comment:
```suggestion
    /// A <see cref="BasicButton"/> class responsible for closing the application.
```
To make the summaries a bit more consistent